.gitignore

```diff
@@ -1,4 +1,8 @@
 # Name of default directory for oscal-cli
 build/oscal-cli
 # VS Code 
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode
```

Comment:
Adding .vscode back in will remove the code snippets/recommended extensions. Is this intentional?

Reply:
totally accidental. will remove

Reply:
Good catch. I think the real issue is that this is ineffectual because .gitignore only prevents new files from being tracked— and .vscode is already tracked.